numberformat_proposed.html

```diff
@@ -721,15 +842,22 @@ <h1>Properties of Intl.NumberFormat Instances</h1>
 
     <ul>
       <li>[[Locale]] is a String value with the language tag of the locale whose localization is used for formatting.</li>
+      <li>[[DataLocale]] is a String value with the language tag of the nearest locale for which the browser has data to perform the formatting operation.  (TODO: is this precisely correct??)</li>
```

Comment:
In TC39, we talk about "implementations" or "JavaScript engines" rather than browsers (e.g., to take Node.js into account). Here's some alternate wording (which also adds a bit of context):

> \[\[DataLocale]] is a String value with the language tag of the nearest locale for which the implementation has data to perform the formatting operation. It will be a parent locale of \[\[Locale]].